src/proc/motion-transform.h

```diff
@@ -59,11 +59,17 @@ namespace librealsense
     class acceleration_transform : public motion_transform
     {
     public:
-        acceleration_transform(std::shared_ptr<mm_calib_handler> mm_calib = nullptr, std::shared_ptr<enable_motion_correction> mm_correct_opt = nullptr);
+        acceleration_transform( std::shared_ptr< mm_calib_handler > mm_calib = nullptr,
+                                std::shared_ptr< enable_motion_correction > mm_correct_opt = nullptr,
+                                bool high_accuracy = false );
 
     protected:
-        acceleration_transform(const char* name, std::shared_ptr<mm_calib_handler> mm_calib, std::shared_ptr<enable_motion_correction> mm_correct_opt);
+        acceleration_transform( const char * name,
+                                std::shared_ptr< mm_calib_handler > mm_calib,
+                                std::shared_ptr< enable_motion_correction > mm_correct_opt,
+                                bool high_accuracy );
         void process_function( uint8_t * const dest[], const uint8_t * source, int width, int height, int actual_size, int input_size) override;
+        bool _high_accuracy = false;
```

Comment:
Please add a comment something like:
`To be refactored and change to accel_scale_factor once we implement sensitivity feature for the accel like the gyro`

Reply:
Done